contents/blog/0-to-hero.md

```diff
@@ -0,0 +1,186 @@
+
+
+## Getting to 1,000 users - do more of what is working, and clarify your ICP
+
+Some users now probably adore your product, others may be indifferent. The key is to figure what is different between the two types. It's [really important](https://newsletter.posthog.com/p/defining-our-icp-is-the-most-important) to [to find your Ideal Customer Profile](creating-ideal-customer-profile).
+
+In practice, we were at a larger user number before we focused on revenue _but_, if I were to repeat this process, I'd have done so sooner as we've changed [how we finance the business](https://posthog.com/handbook/people/finance), so we'd have always had better control of the business. You should do this _way_ earlier if you aren't totally certain you'll be able to make money with your idea.
+
+We started off thinking it'd look like this:
+
+* open source to give away value to the masses, which would generate lots of fans in companies big and small 
+* paid self-hosted version for big companies that didn't want to send data to a 3rd party
+
+This worked fairly well, we started growing quickly. However, at greater we should host it for people in cloud to make money (with a big free tier since cloud is a better experience for the majority of users, and good experience leads to more word of mouth growth).
+
+However, the cloud would have been very competitive for us. We didn't pick it in the first place as we had a lot of unicorn competitors.
+
+To stand out, however, we realized we could focus on building all the products in one (we started as product analytics, but we now have also got session replay, a/b testing, feature flags, web analytics, a data warehouse, a basic customer data platform and user surveys). The key was to not try to also go upmarket so people would have simpler needs. We could not replace Snowflake in a Fortune 500, but we could provide a warehouse for a 30 person startup, with simpler needs - for example.
+
+I did two things to get our very first sales:
+
+1. I talked to users that were already using our product and in our slack community (we've since moved away from Slack to a [custom built forum](https://posthog.com/questions)) about our paid offering and gauging interest
+2. I created a pricing page on our website, listing out our paid extra features and embedding my calendar into it (then gradually making this self serve over time). When someone booked a call, I'd see which extra features they really cared about, and if they wanted to buy.
```

Comment:
```suggestion
1. I talked to users that were already using our product and in our slack community (we've since moved away from Slack to a [custom built forum](https://posthog.com/questions)) about our paid offering and gauging interest.

2. I created a pricing page on our website, listing out our paid extra features and embedding my calendar into it (then gradually making this self serve over time). When someone booked a call, I'd see which extra features they really cared about, and if they wanted to buy.
```